index.html

```diff
@@ -79,8 +79,84 @@
           
         </div>
     </section>
-    
-    
+    <section id="about">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-6 col-lg-8">
+                    <div class="about-heading">
+                        <br>
+                        <br>
+                        <img src="images\about\about.png" alt="about" class="w-100">
+                    </div>
+                    <div class="row px-2 py-3">
+                        <div class=" col-sm-12 col-md-10 col-8">
+                            <br>
+                            <h3 class="about-text">
+                              SheHacks+ VI will give you all the ingredients you need to achieve the ultimate recipe for an award-winning hack!
+                           </h3>
+                        </div>
+                    </div>
+                </div>  
+                <div class="bowl-noodles">
+                    <br>
+                    <br>
+                    <img src="images\about\bowl-noodles.png" alt="about" class="w-100">
+                </div>   
+            </div>
+        </div>
+
+        <div class="container">
+            <div class="tomatoes">
+                <img src="images\about\tomatoes.png" alt="about" class="w-100">
+            </div>   
+            <div class="about_shehacks">
+                <div class="d-flex justify-content-end">
+                    <img src="images\about\shehacks+.png" alt="shehacks+" class="w-50">
+                    <p></p>
+                </div>
+            </div>
+            <div class="row px-2 py-3 d-flex justify-content-end">
+                <div class="  col-sm-8 col-8">
+                    <br>
+                    <h3 class ="text-right about-text">
+                        SheHacks+ is Canada’s biggest all-female and non-binary 36 hour hackathon, which aims to help individuals who are subject to gender discrimination realize they belong in tech. 
+                    </h3>
+                    <br>
+                    <h3 class ="text-right about-text">
+                        Join us and 600 other women+ for a weekend of learning, building, and connecting to build your tech skills and realize that you have allies who are supporting you every step of the way on your tech journey.
+                    </h3>
+                </div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="row d-flex justify-content-end">
+                <div class="col-sm-6 col-lg-8">
+                    <div class="hacker-olympics">
+                        <img src="images\about\hacker-olympics.png" alt="about" class="w-50">
```

Comment:
hacker olympics title is significantly smaller than what is shehacks+ – maybe try w-100?